binary_and_decimal.rb

```diff
@@ -13,9 +23,28 @@ def binary_to_decimal
 #       You may use array indexing to retrieve one element at a time,
 #       compare individual elements with each other and
 #       you may retrieve the length of an array.
-def array_equals(array1, array2)
-  puts "Not implemented"
-  return true
+def array_equals(one, two)
+
+  if one.length != two.length
+    false
```

Comment:
I'd recommend making these explicit returns. i.e. `return false` instead of `false`